src/graph/gremlinEndpoints.ts

```diff
@@ -4,23 +4,13 @@
  *--------------------------------------------------------------------------------------------*/
 
 import { CosmosDBManagementClient } from '@azure/arm-cosmosdb';
-import { nonNullProp, nonNullValue } from '../utils/nonNull';
+import { DatabaseAccountGetResults } from '@azure/arm-cosmosdb/src/models';
+import { nonNullValue } from '../utils/nonNull';
 import { IGremlinEndpoint } from '../vscode-cosmosdbgraph.api';
 
 export async function tryGetGremlinEndpointFromAzure(client: CosmosDBManagementClient, resourceGroup: string, account: string): Promise<IGremlinEndpoint | undefined> {
-    const response = await new Promise((resolve, reject) => {
-        // Use the callback version of get because the Promise one currently doesn't expose gremlinEndpoint (https://github.com/Azure/azure-documentdb-node/issues/227)
-        client.databaseAccounts.get(resourceGroup, account, (error, _result, _httpRequest, innerResponse) => {
-            if (error) {
-                reject(error);
-            } else {
-                resolve(innerResponse);
-            }
-        });
-    });
-
-    const body: string = nonNullProp(<{ body?: string }>response, 'body');
-    const endpointUri = JSON.parse(body).properties?.gremlinEndpoint;
+    const response: DatabaseAccountGetResults = (await client.databaseAccounts.get(resourceGroup, account))._response.parsedBody;
+    const endpointUri = response.documentEndpoint;
```

Comment:
Shouldn't this still be "gremlinEndpoint"?

Reply:
Using callback instead of promise version is not loading any of the graph accounts. Is there a specific use for the gremlinEndpoint? I was thinking given that we are only displaying the account structure in this extension using the .Net SDK endpoint was good enough.

Reply:
You can follow these tests to see how it's used:

https://microsoft.sharepoint.com/teams/azcode/_layouts/OneNote.aspx?id=%2Fteams%2Fazcode%2FSiteAssets%2Fazcode%20Notebook&wd=target%28Testing.one%7CD12CBB51-9002-4371-B958-B88191DF5CF5%2FP3%20Graph%7CC783041F-327A-404C-B04C-DD4AC268B2B6%2F%29
onenote:https://microsoft.sharepoint.com/teams/azcode/SiteAssets/azcode%20Notebook/Testing.one#P3%20Graph&section-id={D12CBB51-9002-4371-B958-B88191DF5CF5}&page-id={C783041F-327A-404C-B04C-DD4AC268B2B6}&end

Reply:
I see what you mean. With the above code I am unable to access the graph. 
Found gremlinEnpoint property within one of the other properties of `response`. Fixed it in the latest commit.